LICENSE

```diff
@@ -198,4 +199,5 @@
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
-   limitations under the License.
\ No newline at end of file
+   limitations under the License.
+   limitations under the License.
```

Comment:
Looks like you accidentally duplicated the last line, other than that the change looks good